frontend/public/components/overview.jsx

```diff
@@ -210,56 +219,128 @@ class OverviewDetails extends React.Component {
     }, groupOptions);
   }
 
-  buildGraphForReplicators(replicators, kind) {
+  addPodsToItem(item) {
     const {pods} = this.props;
-    return _.map(replicators, replicator => {
-      const {uid: replicatorUid} = replicator.metadata;
-      const ownedPods = getOwnedResources(pods.data, replicatorUid);
-      return {
-        ...replicator,
-        kind,
-        pods: ownedPods
-      };
+    return {
+      ...item,
+      pods: getOwnedResources(item.resource, pods.data)
+    };
+  }
+
+  addReplicationControllersToItem(item) {
+    const {replicationControllers} = this.props;
+    const ownedRCs = _.map(getOwnedResources(item.resource, replicationControllers.data), rc => {
+      return this.addPodsToItem({
+        resource: {
```

Comment:
We called this property `obj` in the catalog. I'm OK using either `resource` or `obj`, but I think we should establish a convention if we're going to be following this pattern in a few places. I'd either rename the overview property or the catalog property.

I have a slight preference for `obj` because (to me at least) it's more clear that it's an instance of the thing instead of the just the resource name like `"replicationcontrollers"`.

https://github.com/openshift/console/blob/master/frontend/public/components/catalog.jsx#L60